paper/basic_training.tex

```diff
@@ -765,9 +787,21 @@ \subsubsection{How to choose an appropriate time step?}
 If phase space volume is not preserved, then the sampled ensemble at a later timestep will not be the same as that in which the system was initialized.
 This means that the collected data will not in fact reflect the ensemble of interest.
 Luckily, this issue may be avoided simply by guaranteeing that the integrator is reversible~\cite{Frenkel:2001:}.
-The reason for this outlined in greater detail in \todo[inline, color={green!20}]{JIM: Maybe add citation to Tuckerman paper from 1992}, but the basic idea is that the mathematical operator associated with an integrator must be unitary in order for phase space volume to be preserved.
-By definition of a unitary operator, the application of the operator to negative times is equal to applying the inverse of the operator to an equivalent positive time interval.
-This is in fact also the definition of reversibility: if the operator is applied to propagate forward by $\Delta t$, the starting condition may be recovered by in turn applying the operator to the result using $- \Delta t$ as the timestep.
+The reason for this is outlined in greater detail in ~\citep{Tuckerman:1992:}, but the basic idea is that the mathematical operator associated with an integrator must be unitary in order for phase space volume to be preserved.
+If we define a unitary operator $\mathcal{L}$ that propagates coordinates $\vec{r}(t)$ and velocities $\vec{v}(t)$ (or any function depending on these quantities) over a time interval $\Delta t$
+
+$$\mathcal{L} ( \Delta t, ( \vec{r}(t), \vec{v}(t) ) ) = ( \vec{r}(t + \Delta t), \vec{v}(t + \Delta t) )$$
```

Comment:
I wonder if this is too technical/too many equations for this intro?

Reply:
I would think it would probably be better to make this point without equations, or certainly without as many equations... Or is there something you think the equations add which is vital?